public/swagger/lib/jquery.slideto.min.js

```diff
@@ -0,0 +1 @@
+(function(b){b.fn.slideto=function(a){a=b.extend({slide_duration:"slow",highlight_duration:3E3,highlight:true,highlight_color:"#FFFF99"},a);return this.each(function(){obj=b(this);b("body").animate({scrollTop:obj.offset().top},a.slide_duration,function(){a.highlight&&b.ui.version&&obj.effect("highlight",{color:a.highlight_color},a.highlight_duration)})})}})(jQuery);
```

Comment:
Line is too long.<br>Identifier 'slide_duration' is not in camel case.<br>Identifier 'highlight_duration' is not in camel case.<br>Identifier 'highlight_color' is not in camel case.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'obj' is not defined.